app/src/main/java/com/a0xffffffff/dinesum/User.java

```diff
@@ -42,6 +42,7 @@ public UserState getState()
     public boolean updateState(UserState state)
```

Comment:
What is the UserState for?

Reply:
Oh right I think we decided to remove the user state since we don't really need it. Can @mattwong949  or @skrkasharian confirm that we don't need user state before I remove it?

Reply:
Yes, I think we should remove the 3 UserState classes.